### DISCUSSION TITLE
harbor registry api

### DISCUSSION THREAD

**Discussion by helloxjade** [goharbor/harbor#16440] [Q&A] [answered]
How to obtain a private private registry token by the api ?not the public registry

**Comment by helloxjade** [comment]
i  mean the ip:port

**Comment by wy65701436** [comment, marked as answer]
you can issue an get request with basic auth to the harbor token service, like: 

`GET https://{harbor_ip}/service/token?scope=repository%3Alibrary%2Fhello-world%3Apull&service=harbor-registry`
